modules/tinymce/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Image resize backdrop element did not have `data-mce-bogus="all"` set #TINY-7854
 - Table cells that were both row and column headers would not retain the correct state when converting back to a regular row or column #TINY-7709
 - Clicking beside a non-editable element could cause the editor to incorrectly scroll to the top of the document #TINY-7062
+- Fixed an exception getting thrown when there were more `col` elements than cell columns in a table #TINY-7041
```

Comment:
I'm not too keen on using `cell columns` here as that's not overly clear. I think if anything we should just use `cells` here instead, but would be keen to see what others think or if they have any better suggestions?

Reply:
I don't really like `cell columns` either, but I'd prefer we shorten it to just `columns` instead (as the issue wasn't about more `col` elements than cells in the table). Maybe we could do something like this to specify what sort of "columns" we mean?

```suggestion
- Fixed an exception getting thrown when there were more `col` elements than columns of content in a table #TINY-7041
```

Reply:
Have just removed 'cell' from the sentence which I think should still be clear enough